core/src/main/java/com/klaytn/caver/kct/kip_wrapper/KIP17Wrapper.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2021 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.kct.kip_wrapper;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.kct.kip17.KIP17;
+import com.klaytn.caver.kct.kip17.KIP17DeployParams;
+import org.web3j.protocol.exceptions.TransactionException;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
+
+/**
+ * Representing a KIP17Wrapper
+ * 1. This class wraps all static methods of KIP17
+ * 2. This class should be accessed via `caver.kct.kip17`
+ */
+public class KIP17Wrapper {
+    /**
+     * A Caver instance
+     * this should be injected from KCTWrapper(= `caver.kct`)
+     */
+    private Caver caver;
+
+    /**
+     * Creates a KIP17Wrapper instance
+     * @param caver A Caver instance
+     */
+    public KIP17Wrapper(Caver caver) {
+        this.caver = caver;
+    }
+
+    /**
+     * Deploy a KIP-17 contract.
+     * It must add deployer's keyring in caver.wallet.
```

Comment:
```suggestion
     * The deployer's keyring should be existed in `caver.wallet`.
```

Reply:
@jimni1222 
Thanks for your feedback.
I'll update comments of KIP*Wrappers.